downstairs/src/main.rs

```diff
@@ -1119,6 +1165,10 @@ impl Downstairs {
          * Clear out active jobs, the last flush, and completed information,
          * as that will not be valid any longer.
          *
+         * Don't clear out responses, we need to inform the Upstairs of what
```

Comment:
I'll be back here.

I suspect that telling the old upstairs what we did is not going to help it, really.  We are disconnecting from it, and it can't trust the downstairs when it comes back, so it has to
go through the full reconciliation path anyway.  But, nothing to do here now for this.